exchanges/btcmarkets/README.md

```diff
@@ -25,6 +25,10 @@ Join our slack to discuss all things related to GoCryptoTrader! [GoCryptoTrader
 + REST Support
 + Websocket Support
 
+## Trading Methods Supported
```

Comment:
Did this one die due to the spacing in "Btc markets"?

Reply:
sneaky, will check 🖖

Reply:
[Yes it did](https://github.com/thrasher-corp/gocryptotrader/pull/1619/commits/8342c023428f48f5ccc25e9920e6d86d24d923e2)